src/template.html

```diff
@@ -1,5 +1,5 @@
 <div class="dayparts">
-  <select ng-model="selectedPresetItem" ng-options="item.value as item.label disable when item.disabled for item in presetItems" ng-change="selectedPresetItemChanged()"></select>
+  <select class="form-control" ng-model="selectedPresetItem" ng-options="item.value as item.label disable when item.disabled for item in presetItems" ng-change="selectedPresetItemChanged()"></select>
```

Comment:
This is not a related change to presets. Although, it could be an additional property on the config provider :)